chia/cmds/wallet_funcs.py

```diff
@@ -68,6 +69,23 @@ async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerpri
                 break
 
 
+async def get_csv(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+
+    file_path = args["file"]
+
+    results: dict = await wallet_client.get_csv(args)
+    csv_body: str = results["results"]
+
+    if file_path:
+        f = open(file_path, "w")
+        f.write(csv_body)
+        f.close()
+        print(f"CSV Output to: {file_path}")
+        return
```

Comment:
It's probably cleaner to go with
```python
if file_path:
    # ...
else:
    print(csv_body)

return None
```
i.e. one explit return, and since it's none this function should probably be renamed to `write_csv` or `make_csv`. It's an `async` getter that always returns none..

Reply:
Thanks Nikola! Sorry, weekdays I will be limited due to my primary work. I have tried to follow the conventions I have observed in the existing codebase.  I have modeled my `get_csv` functions after the `get_transactions` functions.  `get_transactions` for example uses the exact same patterns you have critiqued.  Including Returning nothing on a function named `get_*` that is typed to return `-> None`  This function also uses guard clauses style if statements.

Definitions Compared -> get_transactions compared to get_csv:
```python

#wallet.py
def get_transactions_cmd(wallet_rpc_port: int, fingerprint: int, id: int, offset: int, verbose: bool) -> None: 
def get_csv_cmd(wallet_rpc_port: int, fingerprint: int, id: int, offset: int, file: str, date_fmt: str, delim: str) -> None:

#wallet_funcs.py
async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
         async def get_csv(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:

#rpc_wallet_client.py
async def get_transactions(self, wallet_id: str,) -> List[TransactionRecord]: 
         async def get_csv(self, args: dict,) -> Dict:

#rpc_wallet_api.py
async def get_transactions(self, request: Dict) -> Dict:
         async def get_csv(self, args: Dict) -> Dict:
```

These choices were made in an effort to maintain consistency of conventions that currently appear in the code in these files.

That all being said, I have greatly appreciated your help and I will gladly make the suggested changes if we the mods agree that this would be an improvement.  Thanks!

Reply:
It's just suggestions that jump in my eye and it doesn't all have to happen in one PR anyway.

A `get_` functions that is guaranteed to return `None` is a mistake of convention, in my opinion, but it's simpler to stick to conventions if the PR is about something else than cleanup.